pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -584,7 +583,9 @@ func (s *Installer) createValuesOverride(bundle releasev1alpha1.TinkerbellBundle
 				enabled: s.stackService,
 			},
 			relay: map[string]interface{}{
-				enabled: false,
+				enabled:     s.dhcpRelay,
+				image:       bundle.TinkerbellStack.Tink.TinkRelay,
```

Comment:
Are TinkRelay and TinkrelayInit images added by default now or just when dhcp relay is enabled?

Reply:
It is going to be enabled by default. This is also the default behavior of the upstream chart.